CHANGELOG.rst

```diff
@@ -0,0 +1,3 @@
+**0.1.0 - 04/20/23**
```

Comment:
I think its version 1.0.0?

Reply:
Oh this is PRL it dont matter.

Reply:
This is fine for the CODS release, though I think it should be 1.0.0 as Jim said.

We need to add all changes post-CODS to the changelog now as version 1.1.0